datasets/switchboard.py

```diff
@@ -700,11 +700,11 @@ def run(self):
         shutil.copy(glm_file, self.out_glm.get_path())
 
 
-class CreateSwitchboardE2EBlissCorpusJob(Job):
+class CreateSwitchboardSpokenFormBlissCorpusJob(Job):
     """
     Creates a special E2E version of switchboard-1 used for e.g. BPE or Sentencepiece based models.
```

Comment:
Shouldn't this be changed, too?
```suggestion
    Creates a special spoken form version of switchboard-1 used for e.g. BPE or Sentencepiece based models.
```

Also `self.out_e2e_corpus` -> `self.out_spoken_form_corpus `